docs/source/prototype.pipelines.rst

```diff
@@ -53,6 +53,14 @@ HDEMUCS_HIGH_MUSDB_PLUS
    .. autodata:: HDEMUCS_HIGH_MUSDB_PLUS
       :no-value:
 
+HDEMUCS_HIGH_MUSDB_ONLY
```

Comment:
how about renaming it to `HDEMUCS_HIGH_MUSDB`?

Reply:
okay, since other one has the plus, I think that should be fine